.github/workflows/packages.yml

```diff
@@ -28,12 +28,8 @@ jobs:
       - uses: actions/setup-python@v5
         with:
           python-version: "3.10"
-      - name: Install wheel
-        run: pip install --user wheel
-      - name: Building universal wheel
-        run: python setup.py bdist_wheel
-      - name: Building source distribution
-        run: python setup.py sdist
+      - name: Build packages
+        run: ./dev-utils/make-packages.sh
```

Comment:
This should help with reusing the script somewhere else and do some post-build steps